Storage/src/HmacKey.php

```diff
@@ -0,0 +1,217 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Storage;
+
+use Google\Cloud\Storage\Connection\ConnectionInterface;
+
+/**
+ * Represents a Service Account HMAC key.
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Storage\StorageClient;
+ *
+ * $storage = new StorageClient();
+ * $hmacKey = $storage->hmacKey($accessId);
+ * ```
+ */
+class HmacKey
+{
+    /**
+     * @var ConnectionInterface
+     */
+    private $connection;
+
+    /**
+     * @var string
+     */
+    private $projectId;
+
+    /**
+     * @var string
+     */
+    private $accessId;
+
+    /**
+     * @var array|null
+     */
+    private $metadata;
```

Comment:
For consistency with the other classes which use a similar implementation, can we call this `info`?